modules/scalapb-tests/src/main/scala/ru/m2/calypso/scalapb/MissingInstances.scala

```diff
@@ -0,0 +1,46 @@
+package ru.m2.calypso.scalapb
+
+import cats.Eq
+import cats.syntax.eq._
+import com.google.protobuf.timestamp.Timestamp
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.collection.NonEmpty
+import eu.timepit.refined.string.Uuid
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.{Arbitrary, Gen}
+import scalapb.GeneratedEnum
+import scalapb.GeneratedEnumCompanion
+import scalapb.UnknownFieldSet
+import org.scalacheck.Gen.Choose.chooseInstant
+
+import java.time.Instant
+
+object MissingInstances {
+  private val NanosPerMilli    = 1000_000L
+  private val MinTimestampDate = Instant.parse("0001-01-01T00:00:00Z")
```

Comment:
1970-01-01T00:00:00Z

Reply:
Params:
seconds – Represents seconds of UTC time since Unix epoch 1970-01-01T00:00:00Z. Must be from 0001-01-01T00:00:00Z to 9999-12-31T23:59:59Z inclusive.
nanos – Non-negative fractions of a second at nanosecond resolution. Negative second values with fractions must still have non-negative nanos values that count forward in time. Must be from 0 to 999,999,999 inclusive.